### DISCUSSION TITLE
Who has the first firmware version of buds2?

### DISCUSSION THREAD

**Discussion by Dualxs** [timschneeb/GalaxyBudsClient#321] [Q&A] [answered]
please!

**Comment by timschneeb** [comment, marked as answer]
The earliest Buds2 firmware I have backed up is R177XXU0AUH2 (published in August 2021).

Download: https://fw.timschneeberger.me/firmware/download/R177XXU0AUH2 (add '.bin' extension to filename after download)

**Reply by Dualxs** [reply]
thanks bro
